pillar/elife-public.sls

```diff
@@ -26,6 +26,8 @@ elife:
         peterhooper: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDaQaYnpDAaSNkDfuompmMmthxiuqmV8RcDLrfEzQQ8sXrrwxIJyUmO0bhijePxLoOHMZdSz8NcAl2FGYIqWU4bpWpWIhFFUjGBSlN/KjYAhd1eUI+O24JOPPAi5+eInyHS5OJKmqg0MKq6LMbU3zHSFFZdKEfrkECv058X7qtkoMgU7KMtjy9OQZrrA5mWePuTKFrSWV+5Vxhna2L3odURO0ApzqyqsT9jyyTrOl2MFl4c0mFPzdJhZqqa4gjX/eikp8y4cM8C5j3rPvdrxArdTiN2PIuzPDOVYN/2nt8X8tmzTFVfM2/PxdlKINjppjimQH+L0CnpEwhA9IVMBoEF
         hem: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC7gyp8ERD/f3amztI4qVx5oMOU6CrutIhjNrBvXwf98wBOlHvb2bIV72wAplFqrxu8MRrXnNfh0NGT9LkjxQ4E+VKPdDF77jgOPBtrOOwwpGSjzrf/oOHa0bi+ma2Y0bWNQ1WadNPpGdc1L8MMdiB3Hzw3zbLc9AY0biimlHxeWL8T+IhpBmJkT/LKXoMBurOZX/ZbWPzUGkmvqLZrMgqMpM8eOF5JjYmjkwfCKwmBr72AMq4npEOGpgJjO4OlozlbN4yh0BL5CnFDBQrFvMczUbH9JXq5ap1eljYsVaP/bfaV8RzM1OFzORIigrfGGueQsJtuNq+YoEdw6o24SzL7
         tamlyn: ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAIEAmt3C4g/LHecU33xkMCeqzQf4f9VRjRcxCY/GDIl9jwHsJifaAFlfor9xt+i4zz9lCZc8jTcUOH5DikDc3lw0LL7R9cE9i6aedoSb+pWVTBOSSGJIBh6XBWMrnnYYqINIQJIAPGzUn6j9Lceks/2aBn7+U6H5TrwAcZIr8jnK2Is=
+        # tech
+        giancarlo: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDkdTrEimhkRzfny0yGYwC6pi1SATH1I6Dt/0OzoV0rnbS44ALXLGS1+9Z/WKKrZxtgVh5io1umgN9ILadyvFeZQby0XroIXMj43rSYKtaJXYthkf2wL0tmmvrKJnbo11XZRxQY65rN+WfQSvObx0oSuiwiKZUHPFZQpVk18rGb63AdfRgZc09wqlY9qEkmeVv+H2mi73AxTW6V7HfStPJqpKcE1BZyoG3o/flroDdeJXK8yUgJ3HpChlLeCdHPPOjUGahUV5nMOfCo/UDcBligcf6yIxtg8Y3QcXfmYgNnKrrwCVy+KgMjJ20+BpI0ron1OXUHyqUsVqlKFmsp2TTb giancarlofusiello@Ruths-MacBook-Pro.local
```

Comment:
the last value after `ssh-rsa` and `AAAA...` should be removed for simplicity (see other keys)

Reply:
Good spot! Removed.